amplify/backend/function/BorrowFromSchedule/amplify.state

```diff
@@ -0,0 +1,9 @@
+{
+  "pluginId": "amplify-nodejs-function-runtime-provider",
+  "functionRuntime": "nodejs",
+  "useLegacyBuild": true,
+  "defaultEditorFile": "src/index.js",
```

Comment:
Think it should be `src/ts-output/handers/api/BorrowFromSchedule.js` or something?